mlir/test/python/ir/array_attributes.py

```diff
@@ -617,3 +618,119 @@ def test_attribute(context, mview):
     # CHECK: BACKING MEMORY DELETED
     # CHECK: EXIT FUNCTION
     print("EXIT FUNCTION")
+
+
+# CHECK-LABEL: TEST: testGetDenseResourceElementsAttrNdarrayI32
+@run
+def testGetDenseResourceElementsAttrNdarrayI32():
+    class DLPackWrapper:
+        def __init__(self, array: np.ndarray):
+            self.dlpack_capsule = array.__dlpack__()
+    
+        def __del__(self):
+            print("BACKING MEMORY DELETED")
+
+        def get_capsule(self):
+            return self.dlpack_capsule
+
+    context = Context()
+    mview_int32 = DLPackWrapper(np.array([[1, 2, 3], [4, 5, 6]], dtype=np.int32))
+
+    def test_attribute_int32(context, mview_int32):
+        with context, Location.unknown():
+            element_type = IntegerType.get_signless(32)
+            tensor_type = RankedTensorType.get((2, 3), element_type)
+            resource = DenseResourceElementsAttr.get_from_ndarray(
+                mview_int32.get_capsule(), "from_py", tensor_type
+            )
+            module = Module.parse("module {}")
+            module.operation.attributes["test.resource"] = resource
+            # CHECK: test.resource = dense_resource<from_py> : tensor<2x3xi32>
+            # CHECK: from_py: "0x01000000010000000200"
```

Comment:
The line should look something like this I believe:

```suggestion
            # CHECK: from_py: ""0x04000000010000000200000003000000040000000500000006000000""
```